app/javascript/components/helpers/routes.js

```diff
@@ -1,6 +1,7 @@
 export const ALL_STEPS = [
     {id:"1", step:"emotion-selection-web"},
     {id:"1.1.", step:"emotion-entry"},
+    {id:"1.2.", step:"rather-not-say"},
```

Comment:
I do not know the logic of using this constant, but it looks strange to me: some ID`s are with dot in the end id:"1.1." but others are without id:"2.0".  It would be nice to look the same. How do you think?